src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import core.basesyntax.modul.Circle;
+import core.basesyntax.modul.IsoscelesTrapezoid;
+import core.basesyntax.modul.Rectangle;
+import core.basesyntax.modul.RightTriangle;
+import core.basesyntax.modul.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int BOUND = 25;
+    private Random rnd = new Random();
```

Comment:
```suggestion
    private Random random = new Random();
```

it's bad practice to use abbreviations in variable names